docs/sdks/client-side-identity.md

```diff
@@ -176,51 +184,55 @@ This is an example callback which handles the `SdkLoaded` event to call init, an
 
-In order to best support script tags which are not guaranteed to load in order (for example, if you're using `async` or `defer` script tags), you should use the following pattern to register callbacks:
+In order to best support script tags that are not guaranteed to load in order (for example, if you're using `async` or `defer` script tags), use the following pattern to register callbacks:
 
 ```js
 window.__uid2 = window.__uid2 || {};
 window.__uid2.callbacks = window.__uid2.callbacks || [];
 window.__uid2.callbacks.push(callbackFunction);
 ```
 
-This will ensure that:
-- If your code runs before the SDK has loaded (meaning the global `__uid2` object is not available), you can still provide a callback which the SDK can find.
-- If the SDK runs before your code does, you will not overwrite the `__uid2` object or the `callbacks` array.
-- If multiple callbacks are registered using this pattern, they will not over-write each other.
+This ensures the following:
+- If your code runs before the SDK has loaded (meaning the global `__uid2` object is not available), you can still provide a callback that the SDK can find.
+- If the SDK runs before your code does, you do not overwrite the `__uid2` object or the `callbacks` array.
+- If multiple callbacks are registered using this pattern, they do not overwrite each other.
+
+### Provide an Identity to the SDK
 
-### Provide an identity to the SDK
+Unless the SDK is able to load a previously-stored identity from local storage or the cookie, you must provide an identity to it. There are several ways to do this:
 
-Unless the SDK is able to load a previously-stored identity from local storage or the cookie, you will need to provide an identity to it. There are several ways to do this.
+- [Provide an Identity by Setting a First-Party Cookie](#provide-an-identity-by-setting-a-first-party-cookie)
+- [Provide an Identity in the Call to `init`](#provide-an-identity-in-the-call-to-init)
+- [Provide an Identity by Calling `setIdentity`](#provide-an-identity-by-calling-setidentity)
 
-#### Provide an identity by setting a first-party cookie
+#### Provide an Identity by Setting a First-Party Cookie
 
-If you store a first-party cookie as described in the [storage format section](#uid2-storage-format), the SDK will load that value if it is newer than the value available in local storage. If you have set the `useCookie` init option to `true`, it will always load this value and not check local storage. You can control several other things about the cookie using [init parameters](#init-parameters).
+If you store a first-party cookie, as described in the [storage format section](#uid2-storage-format), and the value is newer than the value available in local storage, the SDK loads the value in the cookie. If you have set the `useCookie` init option to `true`, it always loads this value, and does not check local storage. You can control several other things about the cookie using [init parameters](#init-parameters).
```

Comment:
Could we change "the SDK loads the value in the cookie." to "the SDK loads the value from the cookie."?

"Load" can have a slightly different meaning in software which could lead someone to think this means the reverse (i.e. they might reasonably think it means that it will store the value into the cookie). I think using "from" removes any ambiguity.